recipe/meta.yaml

```diff
@@ -24,14 +21,12 @@ requirements:
   host:
     - python
     - pip
-    - openssl
-    - postgresql
     - wheel
-    - krb5  # [win]
+    - setuptools
+    - libpq 12.15
   run:
     - python
     - libpq
```

Comment:
Does `libpq` have a run export for the version? If not, we should probably update this to be `libpq >=X.Y` to match the `host` requirement so users don't accidentally end up using a `libpq` version older than the one we build with.

Reply:
Never mind. I should have just looked at the PostgreSQL recipe to answer it for myself. 😄

Reply:
It does yes It exports `>=12,<13`.